### PR TITLE
release: Bump VERSIONS file to 3.6.0

### PR DESCRIPTION
**Pull request by zvonkok**
Let's bump the VERSIONS file and start preparing for a new release of the project.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1,2 @@
-3.5.0
\ No newline at end of file
+3.6.0
+
```
Comment (gkurz) [inline VERSION:2]: Nit: trailing empty line

```suggestion
```

**Comment by gkurz** [comment]
As expected, commit check is failing :

```
  a7000a3ea335fccfc9b68c4ff3645c84bdac1008    Commit body is missing
```

You can reuse the same text as in the PR description.

```
    Let's bump the VERSIONS file and start preparing for a new release of
    the project.
```

File: VERSION (hunk 1 above)
Comment (zvonkok) [reply VERSION:2]: @gkurz The new line was removed in the last version bump. 

POSIX: 
> 3.206 Line
>A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

We should have a new-line at EOF for every file.

Comment (gkurz) [reply VERSION:2]: I fully agree but this PR is adding two new-line characters :wink:

```
$ od -c VERSION 
0000000   3   .   6   .   0  \n  \n
0000007
```

Comment (zvonkok) [reply VERSION:2]: How hard can it be to bump a VERSION? Zvonko: Yes!

Comment (zvonkok) [reply VERSION]:
```
❯ hexdump -c VERSION 
0000000   3   .   6   .   0  \n                                        
0000006
```

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks Zvonko and Greg!

File: VERSION (hunk 1 above)
Comment (wainersm) [reply VERSION]: > How hard can it be to bump a VERSION? Zvonko: Yes!

I volunteered myself (as you would expect @gkurz ) to do the 3.7.0 release and I start to regret!

**Review by wainersm — APPROVED** [review]

**Pull request merged**